### PR TITLE
ENH: Set and track NumPy's random seed

### PR DESCRIPTION
**Pull request by mgxd**
Continuation of #2086

This PR explicitly sets the numpy random seed.

### CODE REVIEW

**Comment by oesteban** [comment]
Can you split the circleci maintenance into a quickfix to master?

**Review by oesteban — APPROVED** [review]

**Comment by effigies** [comment]
The test should probably be mocked so that the config and environment are reset.

**Comment by mgxd** [comment]
is 008853d what you had in mind @effigies ?

**Comment by oesteban** [comment]
You'll probably need to customize something from:
https://github.com/nipreps/fmriprep/blob/674124ee80b3e2a8affddf005e910e4ca1c97cc0/fmriprep/workflows/tests.py#L10-L35

**Comment by effigies** [comment]
No. You can use mock.patch to mock the environment, and probably the config object, too.

File: fmriprep/tests/test_config.py [hunk 1]
```diff
@@ -51,3 +78,23 @@ def test_config_spaces():
...
+    seeds = config.seeds
+    with patch.dict(os.environ, {}):
+        seeds.load({'_random_seed': master_seed}, init=True)
+        assert getattr(seeds, 'master') == master_seed
+        assert seeds.ants == ants_seed
+        assert seeds.numpy == numpy_seed
+        assert os.getenv("ANTS_RANDOM_SEED") == str(ants_seed)
+
+    _reset_config()
+    assert os.getenv("ANTS_RANDOM_SEED") is None
```
Comment (effigies) [inline fmriprep/tests/test_config.py:98]: Will fail if user has `ANTS_RANDOM_SEED` set.

```suggestion
```

**Review by effigies — APPROVED** [review]

**Pull request merged**